js/exporter/jspdf/export_data_grid_2.js

```diff
@@ -151,85 +50,10 @@ function exportDataGrid(doc, dataGrid, options) {
                 pdfGrid.addRow(currentRow);
             }
 
-            pdfGrid.drawTo(doc);
+            pdfGrid.drawTo(doc, drawPdfTable);
```

Comment:
pdfGrid.drawTo(doc, drawPdfTable); -> pdfGrid.drawTo(doc);

Reply:
fixed it